src/main/webapp/resources/js/pages/user/index.js

```diff
@@ -17,16 +25,18 @@ import UserPasswordPage from "./components/UserPasswordPage";
  */
 render(
   <Provider store={store}>
-    <BrowserRouter basename={setBaseUrl("/users")}>
-      <Routes>
-        <Route path="/:userId" element={<UserAccountLayout />}>
-          <Route index element={<UserDetailsPage />} />
-          <Route path="details" element={<UserDetailsPage />} />
-          <Route path="projects" element={<UserProjectsPage />} />
-          <Route path="password" element={<UserPasswordPage />} />
-        </Route>
-      </Routes>
-    </BrowserRouter>
+    <React.Suspense fallback={<ContentLoading />}>
+      <BrowserRouter basename={setBaseUrl("/users")}>
+        <Routes>
+          <Route path="/:userId" element={<UserAccountLayout />}>
+            <Route index element={<UserDetailsPage />} />
+            <Route path="details" element={<UserDetailsPage />} />
+            <Route path="projects" element={<UserProjectsPage />} />
+            <Route path="security" element={<UserSecurityPage />} />
+          </Route>
```

Comment:
Add a catch all route with the path `*` which would resolve the `<UserDetailsPage />` component. With this you can probably remove this Route `<Route path="details" element={<UserDetailsPage />} />`

Reply:
Please see 5997eee869dfb723ada4c6fc7c1cddaa997131e8.